e2e_test/e2e_transfer_test.go

```diff
@@ -298,6 +301,20 @@ func TestTransferERC20(t *testing.T) {
 			},
 			expectedErr: nil,
 		},
+		{
+			name: "Celo denominated transfer with fee currency",
```

Comment:
Worth adding a test case where `MaxFeeInFeeCurrency` gets exceeded.

Reply:
Agreed. I've now added one that's just -1 below what it should be for acceptance